TodoList.Core/Services/TodoItemService.cs

```diff
@@ -137,6 +137,15 @@ public async Task<IEnumerable<TodoItem>> GetDueTo2DaysItems(ApplicationUser user
                 .ToArrayAsync();
         }
 
+        public async Task<IEnumerable<TodoItem>> GetMonthlyItems(ApplicationUser user, int month)
```

Comment:
You can return an `IList<Tuple<TodoItem, TodoItem>>`, the list will have a maximum of `31` and its item of the list will hold 2 items to be shown in the front-end.

https://docs.microsoft.com/en-us/dotnet/api/system.tuple-2?view=netcore-2.2

Reply:
I'm not completely following here.  Are you talking about returning an IList<Tuple<string, string>>, with the two strings being a field from the Todo entity?